wrappers/md.js

```diff
@@ -2,13 +2,43 @@ import React from 'react'
 import 'css/markdown-styles.scss'
 import Helmet from 'react-helmet'
 import { config } from 'config'
-import { Container, MarkdownContainer, SideBar, StyledLink, SmallTitle } from '../utils/components'
+import { Container, MarkdownContainer, SideBar, StyledLink, Stats, SmallTitle } from '../utils/components'
+import {retrieveMonthlyHoF} from '../utils/fetchData'
 
 export default class Markdown extends React.Component {
+
+  static propTypes = {
+    router: React.PropTypes.object
+  }
+
+  constructor() {
+    super()
+    this.state = {
+      contributors:  [],
+      months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+      isFetchingData: true
+    }
+  }
+
+  componentWillMount() {
```

Comment:
I don't know if best idea is prefetch even if we don't enter Stats page. You will get data when you go for example to Learning page, but you don't need to visit Stats page.